extensions/blocks/revue/revue.php

```diff
@@ -0,0 +1,179 @@
+<?php
+/**
+ * Revue Block.
+ *
+ * @since 8.3.0
+ *
+ * @package Jetpack
+ */
+
+jetpack_register_block(
+	'jetpack/revue',
+	array(
+		'render_callback' => 'jetpack_render_revue_block',
+	)
+);
+
+/**
+ * Revue block render callback.
+ *
+ * @param array $attributes Array containing the Revue block attributes.
+ *
+ * @return string
+ */
+function jetpack_render_revue_block( $attributes ) {
+	if ( ! array_key_exists( 'revueUsername', $attributes ) ) {
+		return '';
+	}
+
+	Jetpack_Gutenberg::load_assets_as_required( 'revue' );
+
+	ob_start();
```

Comment:
Are there any implications for using output buffering rather than creating the output as a string? We should probably settle on an approach because right now we're doing both.